### ISSUE TITLE
Add JUnit output format

### ISSUE THREAD

**Issue by mrded** [mavdi/grunt-cucumberjs#40]
It would be really useful to provide JUnit format without using additional packages such as https://www.npmjs.com/package/cucumber-junit 

JUnit format is required by jenkins.

**Comment by gkushang** [comment]
Are you referring to XML report?

**Comment by mrded** [comment]
Yes, but in special format for [jenkins](https://jenkins-ci.org)

**Comment by mavdi** [comment]
Interesting thing to support. Happy to accept a PR.
